assets/component-facets.css

```diff
@@ -291,6 +320,18 @@
   word-break: break-word;
 }
 
```

Comment:
I wonder if we have a different threshold for colors in terms of show more? For text, 10 makes sense because the list is long, but for color I feel like we could have more rows before showing `show more`?

https://screenshot.click/12-02-15hgf-y47n7.mp4

Reply:
- +1 for using a multiple of 3, it's great to have some rules here to structure the scannability.
- For the limit I'll let @al-mcquiston provide the guidance. 24 seems a lot considering the color labels length and how they wrap on multiple lines.

Reply:
Changed to use 15 for swatch (5 rows), 12 for thumbnail (4 rows)